src/nupic/database/ClientJobsDAO.py

```diff
@@ -55,8 +55,7 @@ def _getLogger():
   needs to be initialized by the host app, and that usually happens after
   imports
   """
-  return logging.getLogger(
-    ".".join(['com.numenta', _MODULE_NAME, ClientJobsDAO.__name__]))
+  return logging.getLogger(__name__)
```

Comment:
@scottpurdy, @SaganBolliger: as far as I know, `logging.getLogger(...)` caches them in a dictionary by logging context name, so it won't create a new logger if one with the same context name already exists.